src/work-allocation-2/containers/work-case-list-wrapper/work-case-list-wrapper.component.ts

```diff
@@ -0,0 +1,275 @@
+import { ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { AlertService, LoadingService } from '@hmcts/ccd-case-ui-toolkit';
+
+import { Caseworker } from 'api/workAllocation2/interfaces/case';
```

Comment:
This is referencing the node layer. We can't import from there - there should be an equivalent in the Angular layer.

Reply:
Created equivalent in Angular layer and updated reference in this file and task-list-wrapper.component.ts